test/Integration/IntegrationTestBase.cs

```diff
@@ -74,35 +76,44 @@ public IntegrationTestBase(ITestOutputHelper output)
         /// </remarks>
         private void SetupDatabase()
         {
-            // Get the test server name from environment variable "TEST_SERVER", default to localhost if not set
-            string testServer = Environment.GetEnvironmentVariable("TEST_SERVER");
-            if (string.IsNullOrEmpty(testServer))
+            SqlConnectionStringBuilder connectionStringBuilder;
+            string connectionString = Environment.GetEnvironmentVariable("AZURE_SQL_DB_CONNECTION_STRING");
```

Comment:
This doesn't really seem to need to be specific to azure, but rather just allow for the user to specify the full connection string. So I'd just call this `SQL_CONNECTION_STRING` or `TEST_CONNECTION_STRING`